src/main/webapp/resources/js/jsfcrud.js

```diff
@@ -0,0 +1,8 @@
+function handleSubmit(args, dialog) {
+    var jqDialog = jQuery('#' + dialog);
+    if (args.validationFailed) {
+        jqDialog.effect('shake', {times: 3}, 100);
+    } else {
+        PF(dialog).hide();
+    }
+}
```

Comment:
This javascript file is pretty small. Should its only method be merged into some existing file?

Reply:
These are questions best suited for #4181 or #2647 and not for #5213.